Lambdas/GHHooks/EventHandler/Handlers/DocsIssuer.swift

```diff
@@ -0,0 +1,135 @@
+import GitHubAPI
+import DiscordBM
+import Logging
+
+/// Sends a "Need translation" message for each PR in a push-commit that needs that.
+struct DocsIssuer {
+
+    enum Configuration {
+        static let docsRepoID = 64560805
+    }
+
+    let context: HandlerContext
+    let commitSHA: String
+    let repo: Repository
+    var logger: Logger {
+        context.logger
+    }
+
+    init(context: HandlerContext) throws {
+        self.context = context
+        self.commitSHA = try context.event.after.requireValue()
+        self.repo = try context.event.repository.requireValue()
+    }
+
+    func handle() async throws {
+        guard repo.id == Configuration.docsRepoID else {
+            return
+        }
+        for pr in try await getPRsRelatedToCommit() {
+            guard isNotExemptFromNewIssue(pr: pr) else {
+                logger.warning(
```

Comment:
This really feels like it should be a `.notice` at most.

Reply:
yeah i guess i'll change to debug